pkg/diagnostics/http_tracing.go

```diff
@@ -93,12 +109,28 @@ func addAnnotationsToSpan(req *fasthttp.Request, span *trace.Span) {
 	})
 }
 
+func isAppChannelServiceInvocationRequest(ctx *fasthttp.RequestCtx, appID, name string) bool {
```

Comment:
why do you need the fasthttp context here?

Reply:
> why do you need the fasthttp context here?

Added earlier to take from userValues from reqCtx but not needed with change. I forgot to remove this param, have removed now. thanks.